src/rtrc.js

```diff
@@ -1305,8 +1305,8 @@ Example:
 						'<h3>' + title + '</h3>' +
 						'<div class="mw-rtrc-diff-tools">' +
 							'<span class="tab"><a id="diffClose">X</a></span>' +
-							'<span class="tab"><a href="' + href + '" target="_blank" id="diffNewWindow">Open in Wiki</a></span>' +
-							'<span class="tab"><a onclick="$(\'.patrollink a\').click()">[mark]</a></span>' +
+							'<span class="tab"><a href="' + href + '" target="_blank" id="diffNewWindow">' + message('openInWiki').escaped() + '</a></span>' +
```

Comment:
Message keys should all be in lowercase.  The intuition commit refers to it as `open-in-wiki`.